### PR TITLE
Refresh nft & collection after mint

### PR DESCRIPTION
**Pull request by dlaxcess**
refresh collection after collection creation & on refresh button click

refresh collection after nft mint & refresh nft after one block after nft mint

### CODE REVIEW

File: svelte/components/Global/Home.svelte [hunk 1]
```diff
@@ -52,19 +57,19 @@
 
       <!-- Select collection -->
       {#if chainId && account}
-        <CollectionList {chainId} {account} bind:address />
+        <CollectionList {chainId} {account} bind:address bind:refreshing />
```
Comment (zapaz) [inline svelte/components/Global/Home.svelte:60]: suppress this binding, while waiting for it in context

**Review by zapaz — CHANGES_REQUESTED** [review]

**Comment by zapaz** [comment]
- NFT description not reseted on 2nd mint
- Select Collection no more active after Mint

**Comment by dlaxcess** [comment]
NFT description not reseted on 2nd mint : RESOLVED
Select Collection no more active after Mint : RESOLVED

**Comment by zapaz** [comment]
Still arrives sometimes... but no that important

**Review by zapaz — APPROVED** [review]

**Pull request merged**